crates/iota-rest-api/src/system.rs

```diff
@@ -453,13 +464,15 @@ impl From<iota_types::iota_system_state::iota_system_state_summary::IotaSystemSt
             safe_mode_non_refundable_storage_fee,
             epoch_start_timestamp_ms,
             epoch_duration_ms,
+            max_committee_members_count,
```

Comment:
This is a field in `AdvanceEpochParams` not in `IotaSystemStateSummaryV2`. Is this  a leftover from a previous iteration?

Reply:
Ok, I see this has been changed with https://github.com/iotaledger/iota/pull/4793/commits/23e1d7acc11951c86f0e854abbf02d01de42642e.

Reply:
Do you plan to refactor `iota-rest-api` and `iota-open-rpc` accordingly?